Makefile

```diff
@@ -69,39 +69,39 @@ build-race: format
 
 build-envoy-asan:
 	@echo "--> building envoy (compilation_mode=fastbuild)"
-	bazelisk build --config=clang-asan //src/envoy:envoy
+	bazel build --config=clang-asan //src/envoy:envoy
```

Comment:
Given that bazelisk is recommended by bazel team, any reason to change to bazel?

Reply:
DONE. Thanks Gopi!